test/fixtures/poro.rb

```diff
@@ -260,9 +262,4 @@ def maker
   cache only: [:id]
   attributes :id
 end
-
-RaiseErrorSerializer = Class.new(ActiveModel::Serializer) do
-  def json_key
-    raise StandardError, 'Intentional error for rescue_from test'
-  end
-end
```

Comment:
The need for this poro was removed in df140293d3010ecbbac50061c22e03a154d1a761  I think there's a bunch of other dead code around like this :)